### PR TITLE
Ft train job clean

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- Gives the train job high resources

### CODE REVIEW

File: k8s/predict/train-job/values-stage.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ app:
   configmap: env-train-job-staging
 
 schedule: 0 1 1 * *
+suspend: true
```
Comment (123MwanjeMike) [inline k8s/predict/train-job/values-stage.yaml:7]: Suspend training in staging. This is done manually as highlighted by @sserurich

File: k8s/predict/train-job/values-prod.yaml [hunk 2]
```diff
@@ -13,6 +14,9 @@ nameOverride: ""
 fullnameOverride: ""
 resources:
   limits:
+    cpu: 8000m
+    memory: 16000Mi
+  requests:
     cpu: 2500m
     memory: 4000Mi
```
Comment (123MwanjeMike) [inline k8s/predict/train-job/values-prod.yaml:21]: Increase resource limits for the train job

**Review by 123MwanjeMike — COMMENTED** [review]
GUiding comments

File: k8s/predict/train-job/values-stage.yaml (hunk 1 above)
Comment (Baalmart) [reply k8s/predict/train-job/values-stage.yaml:7]: Thanks @123MwanjeMike , does this mean that the manual process with partly continue...in staging? Please shed some more light on this. Thanks.

Comment (123MwanjeMike) [reply k8s/predict/train-job/values-stage.yaml:7]: Yes @Baalmart 
The automated training of the model is not a requirement for the staging environment as highlighted by @sserurich

**Review by Baalmart — COMMENTED** [review]
This LGTM, thanks @123MwanjeMike

**Review by Baalmart — APPROVED** [review]

**Pull request merged**